src/ResourceManager/AnalysisServices/Commands.AnalysisServices/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.Analysis" version="2.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Analysis" version="2.0.3" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
Remove the last three packages added - these are in common dependencies and not needed here.

Reply:
done